ephios/core/forms/users.py

```diff
@@ -52,6 +52,7 @@
     "core.add_qualification",
     "core.change_qualification",
     "core.delete_qualification",
+    "auth.publish_event_for_group",
     "modellogging.view_logentry",
 ]
```

Comment:
You could check of 

MANAGEMENT_PERMISSIONS zu CORE_MANAGEMENT_PERMISSIONS umbenennen (reines refactoring)

right here